### ISSUE TITLE
Image Save node - Missing outputs in API

### ISSUE THREAD

**Issue by robballantyne** [WASasquatch/was-node-suite-comfyui#343]
Hi

Very quick non-urgent issue I have come across today.

When using the save image node with `show_previews: false` there is no entry for output images in the /history endpoint

Naturally the fix is to enable previews, but something to consider if you get the time.

Cheers!
Rob

**Comment by WASasquatch** [comment]
Yeah that probably doubles as both. Since if there is output, the node shows a preview. So can't have both.
